webpack.config.js

```diff
@@ -42,7 +31,12 @@ module.exports = {
       },
     ],
   },
-  externals: nodeModules,
+  externals: [
+    nodeExternals({
+      modulesFromFile: true,
+      whitelist: [/^@babel\/runtime\//],
```

Comment:
@EnTeQuAk in the web-ext package we do have @babel/runtime as a dependency (from what I recall we did choose to add it as a direct dependency because is technically something that is being used in production), do you have anything against moving it from the devDependencies to the dependencies for the addons-linter too?

In my opinion moving it to the dependencies makes it clear that it is being used, much more clear than whitelisting it in the webpack config here.

If we have reasons that makes us to prefer keeping that unmodified, then I'm fine with whitelisting it here.

Reply:
I don't have any reason against it, so let's add it to `dependencies`.